tests/topotests/bgp_unnumbered/r1/bgpd.conf

```diff
@@ -0,0 +1,12 @@
+!
+router bgp 65001
+  timers bgp 1 9
+  no bgp ebgp-requires-policy
+  neighbor r1-eth0 interface remote-as external
+  address-family ipv4 unicast
+  exit-address-family
+ !
+!
+route-map r2 permit 10
+  set community blackhole
```

Comment:
Where is this used?

Reply:
it's not I cut-n-pasted from another test.  I'll just remove